content/show/this-week-in-bitcoin/0062.md

```diff
@@ -0,0 +1,35 @@
+{
+  "type": "episode",
+  "draft": false,
+  "show_slug": "this-week-in-bitcoin",
+  "show_name": "This Week In Bitcoin",
+  "episode": 62,
+  "episode_padded": "0062",
+  "episode_guid": "382338f4-6a14-4f88-8329-a8685f22d075",
+  "slug": "62",
+  "title": "Bitcoin Is the Geiger Counter",
+  "description": "Bitcoin doesn't lie about world events, and this week proved it. Plus, my thoughts on Bitcoin Core facing its own flavor of cancel culture and why Arthur Hayes predicts $9 trillion will be printed by 2028.",
+  "date": "2025-06-25T18:26:56",
+  "header_image": "/images/shows/this-week-in-bitcoin.png",
+  "categories": [
+    "This Week In Bitcoin"
+  ],
+  "tags": [],
+  "hosts": [],
+  "guests": [],
+  "sponsors": [],
+  "podcast_duration": "01:00:22",
+  "podcast_file": "https://feeds.fountain.fm/40huHEEF6JMPGYctMuUI/items/tPIJU5ZQ7IZmmh6ltJt8/files/AUDIO---DEFAULT---8f12d5a6-f97b-4528-be9c-49e754968dc7.mp3",
+  "podcast_bytes": 87407925,
+  "podcast_chapters": null,
+  "podcast_alt_file": null,
+  "podcast_ogg_file": null,
+  "video_file": null,
+  "video_hd_file": null,
+  "video_mobile_file": null,
+  "youtube_link": null,
+  "jb_url": "https://www.jupiterbroadcasting.com/show/this-week-in-bitcoin/62",
+  "fireside_url": null,
+  "value": null
+}
+
```

Comment:
![image](https://github.com/user-attachments/assets/3fbfa43b-c247-4535-b9e9-45c986e1060b)

Reply:
Yeah alas no `<link>` in the source feed at the item level. So all we've got is JB url via number...